src/kixi/stats/distribution.cljc

```diff
@@ -506,6 +506,59 @@
         :cljs (ISeqable
                (-seq [this] (sampleable->seq this)))))
 
+(deftype ^:no-doc Cauchy
+  [location scale]
+  p/PRandomVariable
+  (sample-1 [this rng]
+    (+ location (* scale (tan (* PI (- (rand-double rng) 0.5))))))
+  (sample-n [this n rng]
+    (default-sample-n this n rng))
+  p/PQuantile
+  (cdf [this x]
+    (+ 0.5 (/ (atan (/ (- x location) scale)) PI)))
+  (quantile [this p]
+    (+ location (* scale (tan (* PI (- p 0.5))))))
+  #?@(:clj (clojure.lang.Seqable
+            (seq [this] (sampleable->seq this)))
+      :cljs (ISeqable
+             (-seq [this] (sampleable->seq this)))))
+
+(deftype ^:no-doc Lognormal
+  [logmu logsd]
```

Comment:
I'd prefer to have `mu` and `sd` parameter names here

Reply:
The reason I used `:logmu` and `:logsd` was the following text

> These are the expected value (or mean) and standard deviation of the variable's natural logarithm, not the expectation and standard deviation of {\displaystyle X}X itself.


on this [Wikipedia entry](https://en.wikipedia.org/wiki/Log-normal_distribution#Generation_and_parameters).

Just to make it clear that the parameters are not the same as a Normal distribution.

Reply:
The parameters are conventionally denoted as 𝜇 / mu and σ2 / sd, so I find those the least surprising way to parameterise the distribution (and the helper `log-normal` function), even though I follow your reasoning. I don't see a contradiction: they are the same parameters as the normal distribution, but in a different context, much like the shape, scale and rate parameters for other distributions.

Reply:
Okay.  I'll change the names and add to the docstring to specify what the parameters refer to.